examples/sierpinski.js

```diff
@@ -1,9 +1,10 @@
-// draw a Sierpinski Curve of arbitrary recursive depth
 // https://en.wikipedia.org/wiki/Sierpi%C5%84ski_curve
 
+/** helper for `sierpinski` */
 function halfSierpinski(size, level) {
-   if (level <= 0)
+   if (level <= 0) {
       forward(size);
```

Comment:
Alternatively, rather than add a pair of curly braces, just move `forward(size);` to the same line as `if (level <= 0)`:
`if (level <= 0) forward(size);`

Reply:
Seems like a good idea. I added braces just for consistency with the `else`, and because it's usually good practice to have braces whenever there's an `else` (regardless of number of statements), to reduce the likelihood of confusing dangling-`else`s. But I guess we can make an exception for this rule, since there's no nested `if`s

Reply:
Wait! I got an idea. Let's use a guard clause to get rid of the `else`